src/FINDOLOGIC/Export/CSV/CSVExporter.php

```diff
@@ -28,7 +28,13 @@ public function __construct($itemsPerPage, $propertyKeys)
      */
     public function serializeItems($items, $start = 0, $count = 0, $total = 0)
     {
-        $export = self::HEADING;
+        // To enable pagination, don't write the heading if it's anything but the first page.
```

Comment:
Add `$export = '';` on top and only write heading if `$start === 0`

Reply:
:heavy_check_mark: